package.json

```diff
@@ -29,30 +29,30 @@
   ],
   "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.2.0",
-    "ember-cli": "^2.7.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.7.0",
```

Comment:
With `ember-cli` specifically you do no want to do this.  When changing the versions of Ember CLI used it's not enough to just change the version number in *package.json* but you must also run `ember init`, which may nor may not (depending on what changes are in the release) introduce other required code changes in the codebase.  Point being, a newer version of Ember CLI, which a caret would install, may require associated changes to the codebase that were not made, causing the application to not be able to be dev-ed, built, etc.   We could possibly consider the use of `~` but I would want to review the change logs of the prior patch releases of Ember CLI before feeling comfortable doing so.